sda-download/.github/integration/tests/s3notls/52_check_endpoint.sh

```diff
@@ -70,12 +70,21 @@ echo "expected file found"
 # Test file can be decrypted
 ## test also the files endpoint
 
-C4GH_PASSPHRASE=$(grep -F passphrase config.yaml | sed -e 's/.* //' -e 's/"//g')
+C4GH_PASSPHRASE=$(grep -F passphrase config.yaml | tail -1 | sed -e 's/.* //' -e 's/"//g')
```

Comment:
```suggestion
C4GH_PASSPHRASE=$(grep -F passphrase config.yaml | sed -e 's/.* //' -e 's/"//g')
```

Reply:
I think it's a valid change in case there are multiple lines with the word `passphrase`. Furthermore, we may need to refactor it when SDA supports multiple keys.

Reply:
changed all yaml parsing to using `yq` as suggested above